R/tspa.R

```diff
@@ -113,27 +114,50 @@
 
 
 tspa <- function(model, data, reliability = NULL, se = NULL,
-                 vc = NULL, cross_loadings = NULL, ...) {
+                 fsT = NULL, fsL = NULL, ...) {
+
+  if (nchar(model) == 0) {
```

Comment:
I'm thinking whether we need `model` to also be character, as `nchar(12)` for example would be 2